### ISSUE TITLE
Not aligned after smoothing

### ISSUE THREAD

**Issue by STK2666** [fudan-generative-vision/champ#158]
After smoothing the smpl results will be not aligned.
<img width="128" alt="微信图片_20240909110633" src="https://github.com/user-attachments/assets/9ffdd7d9-1504-47c1-8e24-eab1acda0af3">
<img width="128" alt="微信图片_20240909110642" src="https://github.com/user-attachments/assets/e55aa624-014d-4c2f-a5d8-0b0e4533b513">
And my code is:
```python
def export_smpls_group_npy(dir_path):
    smpls_save_path = join(dir_path, 'new_smpls')
    os.makedirs(smpls_save_path, exist_ok=True)
    obj = bpy.data.objects[OBJ_NAME]
    arm_ob = bpy.data.objects[ARM_OBJ_NAME]
    scene = bpy.data.scenes['Scene']
    obname = "m_avg"

    for frame in range(scene.frame_start, scene.frame_end):
        smpl, cam = reverse_blender_para_to_smpl(obj, arm_ob, obname, bpy.data.scenes['Scene'], frame=frame)
        betas = torch.tensor(smpl['betas'], dtype=torch.float32)
        global_orient = torch.tensor(smpl['global_orient'], dtype=torch.float32)
        body_pose = torch.tensor(smpl['body_pose'], dtype=torch.float32)
        smpl_output = model.smpl(betas=betas, global_orient=global_orient, body_pose=body_pose, pose2rot=False)
        smpl_outs = {
            'global_orient': smpl['global_orient'],
            'betas': smpl['betas'],
            'body_pose': smpl['body_pose'],
        }
        pred_vertices = smpl_output.vertices.reshape(1, -1, 3)
        all_verts = []
        all_cam_t = []
        for n in range(pred_vertices.shape[0]):
            verts = pred_vertices[n].detach().cpu().numpy()
            all_verts.append(verts)
            all_cam_t.append(cam)

        img_size = 256.0

        scaled_focal_length = np.array(5000.0)

        results_dict_for_rendering = {
            "verts": all_verts,
            "cam_t": all_cam_t,
            "render_res": np.array([img_size, img_size]),
            "smpls": smpl_outs,
            "scaled_focal_length": scaled_focal_length,
        }

        frame_save_path = join(smpls_save_path, f"frame_{frame:05d}.npy")
        np.save(frame_save_path, results_dict_for_rendering)
```
Could anyone told me what is wrong?

**Comment by STK2666** [comment]
I found I forgot to reverse the Y-axis. It is solved now.
```python
for n in range(pred_vertices.shape[0]):
            verts = pred_vertices[n].detach().cpu().numpy()
            all_verts.append(verts)
            cam[1] *= -1
            all_cam_t.append(cam.astype(np.float32))
```

https://github.com/user-attachments/assets/1963e45f-917e-4ca6-bbf4-1d7c0c64ff57
